sql/moz-fx-data-shared-prod/mozilla_org_derived/gclid_conversions_v2/query.sql

```diff
@@ -33,18 +36,61 @@ dl_token_to_telemetry_id AS (
   FROM
     `moz-fx-data-shared-prod.telemetry_derived.clients_first_seen_v2`
 ),
-telemetry_id_to_activity AS (
+--Step 4: Get all clients who were active on the activity date, and the type of activity they had
+telemetry_id_to_activity_staging AS (
   SELECT
     client_id AS telemetry_client_id,
     submission_date AS activity_date,
     search_count_all > 0 AS did_search,
     ad_clicks_count_all > 0 AS did_click_ad,
+    NULL AS first_wk_5_actv_days_and_1_or_more_search_w_ads,
```

Comment:
Safer to cast to the correct data type for all the NULLS,
```suggestion
    CAST(NULL AS BOOLEAN) AS first_wk_5_actv_days_and_1_or_more_search_w_ads,
```